.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -0,0 +1,65 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug", "triage"]
+# assignees:
+#   - onesignal/unity
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what steps are required to reproduce.
+      placeholder: Tell us what's going on.
+      value: "Something isn't quite right!"
+    validations:
+      required: true
+  - type: textarea
+    id: what-are-expectations
+    attributes:
+      label: What did you expect to happen?
+      description: Also tell us, what did you expect to happen?
+      placeholder: Tell us what you see!
+      value: "A bug happened!"
+    validations:
+      required: true
+  - type: dropdown
+    id: unity-version
+    attributes:
+      label: Unity version
+      description: What version of Unity are you running?
+      options:
+        - Unity 2021.2.12 (latest)
+        - Unity 2021.x
+        - Unity 2020.x
+        - Unity 2019.x
+    validations:
+      required: true
+  - type: dropdown
```

Comment:
Could this be a check box so they can select both? If not can we add another options for "Android & iOS"?

Reply:
Fixed in https://github.com/OneSignal/OneSignal-Unity-SDK/pull/454/commits/bd9d3ce5615eb6037fd07d0048fff1291f77a6f6.